pkg/fission-cli/main.go

```diff
@@ -260,6 +260,7 @@ func NewCliApp() *cli.App {
 	pkgDeployArchiveFlag := cli.StringSliceFlag{Name: "deployarchive, deploy", Usage: "Local path or URL for binary archive"}
 	pkgBuildCmdFlag := cli.StringFlag{Name: "buildcmd", Usage: "Build command for builder to run with"}
 	pkgOutputFlag := cli.StringFlag{Name: "output, o", Usage: "Output filename to save archive content"}
+	pkgStatusFlag := cli.StringFlag{Name: "status", Usage: `filter packages status succeeded or failed"`}
```

Comment:
Currently, fission supports not only succeeded/failed states. 
https://github.com/fission/fission/blob/b18462b10b600fe2de47c523a5e256039757f9f5/pkg/apis/fission.io/v1/const.go#L44-L48
So let's change the description a bit
```
Package status to filter packages  
```

Reply:
Is it better to use this statement?
```
Package status to filter packages
```

Reply:
I think what i mean here is to filter the packages by the package status, how about we change it to Filter package by status

Reply:
That's fair. Let's do it. nitpick: `packages` -> `Filter packages by status`